code/BombPro/zutils.ino

```diff
@@ -166,4 +166,40 @@ void activateRelay(){
 
 }
 
+char *centerText (char *text, char *wrapper) {
+  /*
+   * The purpose of this function is to return text in a centered fashion.
+   * The default wrapper is ==
+   */
+  const uint8_t width = 20; // width of the lcd panel
+  bool useWrapper = true;
+  if (strlen(text) > width ) {
+    // text is just too long
+    return text;
+  } else if (strlen(text) + (strlen(wrapper) * 2) > 20 ) {
+    // text would be too long with the wrapper
+    useWrapper = false;
+  }
+
```

Comment:
should add a check if the wrapper is empty string and then set useWrapper to false